configs/dense_teacher/dense-teacher_fcos_r50-caffe_fpn_gn_180k_semi-0.1-coco.py

```diff
@@ -0,0 +1,82 @@
+_base_ = [
+    './fcos_r50-caffe_fpn_gn-head-center-normbbox-centeronreg-giou_1x_coco.py',
+    '../_base_/default_runtime.py',
+    '../_base_/datasets/semi_coco_detection.py',
```

Comment:
The data augmentation of `semi_coco_detection` cannot be directly applied to `DenseTeacher`,  because of the geometric displacement and lack of  `GaussianBlur` .